packages/cli/tsup.config.js

```diff
@@ -13,7 +13,7 @@ export default defineConfig({
   sourcemap: true,
   clean: true,
   noExternal: Object.keys(packageJSON.dependencies).filter(
-    f => !excludedPackages.includes(f)
+    (f) => !excludedPackages.includes(f)
```

Comment:
It seems that if I use either: 

1. vscode prettier
2. `npx prettier --fix`
3. `npx eslint --fix` 

They all are formatting the code in different ways, is this known, or a problem with my local environment?